src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/custom.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def cli_billing_list_invoices(client, generate_url=False):
+    '''List all available invoices of the subscription'''
+    invoices = client.list(expand='downloadUrl' if generate_url else None)
+    return list(invoices)
```

Comment:
i choose not use sdk reflection of list functions in billing, because it will expose all parameters in sdk, which i dont want to do that in CLI.